src/components/Button/Button.tsx

```diff
@@ -19,44 +15,14 @@ export interface Props {
    * Disables the field and prevents editing the contents
    */
   disabled?: boolean;
-  /**
-   * Toggles button that fills the full width of its container
-   */
-  fullWidth?: boolean;
-  /**
-   * Visually hide button text (but text is still accessible to assistive technology)
-   */
-  hideText?: boolean;
   /**
    * Link to URL. If href is present, the button will be rendered as an <a> element.
    */
   href?: string;
-  /**
-   * Name of SVG icon (i.e. caret-down, minus, warning)
-   */
-  iconName?: string;
-  /**
-   * Determines position of icon relative to button text.
-   * - **before** places icon before button text
-   * - **after** places icon after button text
-   */
-  iconPosition?: 'before' | 'after';
-  /**
-   * Button rendered on a dark backgorund
-   */
-  inverted?: boolean;
-  /**
-   * Loading state passed down from higher level used to trigger loader and text change
-   */
-  loading?: boolean;
```

Comment:
I actually really like this paradigm of having a spinner in the button and would be happy to keep this one if design feels it's a solution they want to start using. Or maybe we're already doing it and we want to expose it in the design system. In any case, I'd like design to weigh in on this one.

Reply:
@dierat yep, this is related to the icon conversation above. It depends on how much you want to enforce a consistent loading button state at an EDS level vs leaving it to downstream teams to implement things in a way that suits them best.

Reply:
I'll revert this change and ask design whether we want to support this feature in this component: https://app.shortcut.com/czi-edu/story/188471/button-api-questions-for-design